src/Tickets/Commerce/Order.php

```diff
@@ -408,11 +417,40 @@ static function ( $item ) {
 			},
 			$items
 		) );
+
+		$subtotal = $this->get_value_total( $items );
+
+		$original_cart_items = $items;
+
+		/**
+		 * Filters the cart items before creating an order.
+		 *
+		 * Allows modification of the cart items array before creating an order,
+		 * passing the current subtotal, payment gateway, and purchaser details.
+		 *
+		 * @since TBD
+		 *
+		 * @param array             $items     The items in the cart.
+		 * @param Value             $subtotal  The calculated subtotal of the cart items.
+		 * @param Gateway_Interface $gateway   The payment gateway used for the order.
+		 * @param array|null        $purchaser An array of purchaser information including purchaser_user_id,
```

Comment:
Same as above if possible update the doc block like [this](https://github.com/the-events-calendar/event-tickets/blob/7ca697bedf568ba7de101311a9dd8fc98b42d402/src/Tickets/Seating/Tables/Sessions.php#L283-L287).

Reply:
@rafsuntaskin The difference here is that these are separate variables passed to the filter rather than one array.